### PR TITLE
feat: add vue addon

### PR DESCRIPTION
**Pull request by rafaelramalho19**
One of our latest projects uses Vue as the front-end framework.
Therefore, I isolated the main configs into this repo.

Please acknowledge that Vue has standard conventions different from React and other libraries.

### CODE REVIEW

File: test/fixtures/rules/vue/vue.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// `vue/html-quotes` - enforce double quotes
```
Comment (acostalima) [inline test/fixtures/rules/vue/vue.js]: Vue's ESLint plugin [has many, many more rules](https://eslint.vuejs.org/rules/). Shouldn't we add fixtures for other key rules as well? 🤔

**Review by acostalima — CHANGES_REQUESTED** [review]
🙌

File: test/fixtures/rules/vue/vue.js (hunk 1 above)
Comment (rafaelramalho19) [reply test/fixtures/rules/vue/vue.js]: It does 😅 
Guess I'll just stop being lazy and do them all

Comment (rafaelramalho19) [reply test/fixtures/rules/vue/vue.js]: Done 😓

**Review by satazor — APPROVED** [review]

**Comment by satazor** [comment]
PR is good, but we need to add this new addon to the README.

**Pull request merged**